ChangeLog

```diff
@@ -1,3 +1,7 @@
+2015-06-02  Sherine Awad  <drmahmoud@ucdavis.edu>
+   * Fix minor issues in split-paired-reads 
```

Comment:
Could you describe in a bit more detail what you've done? Also, please put a space between your name and the first line of changes.